appcenter-analytics/ios/AppCenterReactNativeAnalytics.xcodeproj/project.pbxproj

```diff
@@ -232,10 +232,16 @@
 			buildSettings = {
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(SRCROOT)/../../../ios/Vendor/**",
-					"$(SRCROOT)/../../../ios/Pods/**",
+					"$(SRCROOT)/../../../ios/Pods/AppCenter/**",
+					"$(SRCROOT)/../../../ios/Pods/AppCenterReactNativeShared/**",
 					"$(inherited)",
 				);
-				HEADER_SEARCH_PATHS = "$(SRCROOT)/../../react-native/React/**";
+				HEADER_SEARCH_PATHS = (
+					"$(SRCROOT)/../../react-native/React/Base",
```

Comment:
Why narrowing down the header search paths from ** to 4 specific modules?

Reply:
We can't have recursive at root as it causes https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/246. So we try to limit to what we need.